modules/internal/ChapelSyncvar.chpl

```diff
@@ -280,7 +280,7 @@ module ChapelSyncvar {
     return wrapped.isFull;
   }
 
-  proc   = (ref lhs : _syncvar(?t), rhs : t) {
+  operator =(ref lhs : _syncvar(?t), rhs : t) {
```

Comment:
Given that we made Atomic operators methods, I'd probably make the sync ones methods too.

Reply:
Eh, actually these are deprecated, so probably not worth the time to change.